lib/store/settings_store.dart

```diff
@@ -288,6 +299,9 @@ abstract class SettingsStoreBase with Store {
         sharedPreferences.getInt(PreferencesKey.displayActionListModeKey) ??
             defaultActionsMode));
     var pinLength = sharedPreferences.getInt(PreferencesKey.currentPinLength);
+    final pinCodeTimeOutDuration = PinCodeRequiredDuration.deserialize(raw: sharedPreferences.getInt(PreferencesKey.pinTimeOutDuration) 
+      ?? defaultPinCodeTimeOutDuration);
```

Comment:
and here, since we will change the `defaultPinCodeTimeOutDuration` type so I'd say to check if the value from shared pref is null or not